tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -10,12 +10,12 @@ exports[`declare.php 1`] = `
     </body>
 </html>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-<?php
-declare(strict_types=1);
-?>
+<?php declare(strict_types=1); ?>
 <html>
     <body>
-        <?php $a = 1; ?>
+        <?php
```

Comment:
should be `<?php $a = 1; ?>`, need to figure that out

Reply:
So I looked into it and this is a muuuuch bigger issue :(

The reason this happens is because we use the raw inline value when printing, and prettier thinks the whole start of the line is

```html
<html>
    <body>
         <?
```
, rather than just the current line we're on (ie `    <?`). To get this to work, we would need to use `join(hardline, node.raw.split("\n")` instead of just `raw`, but this causes a ton of other problems because we call `removeNewLines` which strips out any `hardlines` (the `raw` value gets around that currently)

Reply:
cc @evilebottnawi @czosel

Reply:
I was hoping there would be a way to signal to prettier that we're currently at a certain offset/column when printing, but I couldn't find anything

Reply:
@mgrip let's merge this PR as is and do it in other PR, i have idea